src/api/dataFetching.js

```diff
@@ -101,57 +42,6 @@ export default {
     return parseImages(data);
   },
 
-  // Voir si en utilisant le full text search avec l'id c'est mieux
-  async getRelatedImages(tag, id) {
```

Comment:
do you still have the code if we need to regenerate the website json files (tags, decade, etc.)

Reply:
Yes, I have it, I can create a tools folder for this purpose.

Reply:
yes, sounds good to make sure we have the code for that